src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  }
+
+  if (days >= 3) {
+    return days * 40 - 20;
+  }
+
+  return days * 40;
```

Comment:
It might be better to not calculate just certain cases with certain numbers, because values can change and all functionality needs to rewrite.

Reply:
sure. i've added variables 👍